src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -426,16 +426,38 @@ has to be updated, then for MEG it could be considered to be a new session.
 
 ## Empty-room MEG recordings
 
-Empty-room MEG recordings capture the environment and system noise. Their
-collection is RECOMMENDED, before/during/after each session. This data is stored
-inside a subject folder named `sub-emptyroom`. The `session label` SHOULD be
-that of the date of the empty-room recording (e.g. `ses-YYYYMMDD`). The
-`scans.tsv` file containing the date/time of the acquisition SHOULD also be
-included. Hence, users will be able to retrieve the empty-room recording that
-best matches a particular session with a participant, based on date/time of
-recording.
+Empty-room MEG recordings capture the environmental and recording system's
+noise.
+There are two widely used strategies for collecting these recordings:
 
-Example:
+1.  Some labs record one empty-room measurement per day, which is then shared by
+    all researchers that record subject data on that day
+
+1.  Some labs make it the responsibility of each researcher to record a short
+    empty-room measurement prior to each subject-specific recording
+    (e.g., while preparing the study subject for the task)
```

Comment:
is there any particular reason we want to store this differently? My argument against this would be that it would lead to more branching paths in software. You'd need to consider both the cases. Ultimately to use BIDS data, researchers have to use BIDS-specific software, so you're increasing the burden on these software for no added benefit.

Reply:
Yes, there are lab and experimental design specific reasons for recording this differently and hence also for storing this differently. To name a few:

- if the noise varies a lot over the day (e.g. due to activities around the building, like the MEG system being close to the parking lot which has a lot of activity at the start and end of the day)
- if potentially noisy experimental equipment is introduced in the shielded room for some experiments, but not others  
- if some researchers measure their participants in a supine position, and others in a seated position
- if the settings of the MEG recording are different over experiments (e.g. different sampling rates, with or without active shielding, etc.) 

If the MEG system hardware itself is stable (more so for CTF systems and newer Neuromag systems than for older Neuromag systems), the environment has constant noise, everyone records with the same dewar position and settings, uses the same stimulus equipment, and/or if you are specifically interested in more slowly varying MEG system (squids and electronics) noise characteristics than environmental noise, then it makes sense to only do a daily or weekly recording.

Reply:
sure I can see why you would want two strategies for measuring empty room noise. But I don't see why that should necessarily translate to storing in two different folders. You could still get the corresponding empty-room file for a certain measurement by reading in the dates from the file metadata. If we want to encourage linking via BIDS naming conventions, then we need to make more explicit recommendations how the files should be named in strategy 2. Beyond saying they should be stored in the same folder.

Reply:
I find this quite simple: subject- and session-specific measurements (of noise, or whatever) are stored within the same session (sub-001), measurements that are not done in the same session, and that are nor session/subject-specific are stored elsewhere (sub-noise).

In most data that we share the recording date is actually not shared due to privacy reasons. If noise measurements are required to be linked through dates to the subject measurements, we (i.e. the Donders), would simply not share the noise data to protect the privacy of our subjects.

Reply:
Yes, but in BIDS when you anonymize you will shift the date instead of removing it. So you'd do this for the entire study including the noise measurements.

Anyway, I can live with this even though I don't see a good argument. @hoechenberger you'd need to implement this at some point in `mne-bids` :)